bundles/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/internal/handler/SqueezeBoxServerHandler.java

```diff
@@ -306,6 +331,8 @@ public void login() {
         if (StringUtils.isEmpty(userId)) {
             return;
         }
+        // Create basic auth string for jsonrpc interface
+        basicAuthorization = new String(Base64.getEncoder().encode((userId + ":" + password).getBytes()));
```

Comment:
You should specify the charset used here.

Reply:
Ok. Done. BTW there are quite a few places in the code base that don't specify the charset.